lib/core_extensions/hash.rb

```diff
@@ -0,0 +1,15 @@
+module CoreExtensions
```

Comment:
If this is borrowed from Rails, please put a comment at the top with the original source.

Reply:
Turns out this is part of ruby starting from 2.4.6: [docs](https://apidock.com/ruby/v2_4_6/Hash/transform_values). For prior versions, Rails extends the `Hash` class starting from [4.2.1](https://apidock.com/rails/v4.2.1/Hash/transform_values).